crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -1050,17 +1047,14 @@ end
 #! Panics if:
 #! - the prefix or suffix of the provided foreign account ID equal zero.
 #! - the maximum allowed number of foreign account to be loaded (64) was exceeded.
-export.get_foreign_account_ptr
+export.get_account_data_ptr
```

Comment:
We should also update doc comments above (e.g., lines 1036 - 1043 above). But we can do it in a follow-up PR.